NumLib/Extrapolation/LocalLinearLeastSquaresExtrapolator-impl.h

```diff
@@ -83,10 +83,10 @@ extrapolateElement(std::size_t const element_index,
     for (auto int_pt=decltype(ni){0}; int_pt<ni; ++int_pt)
     {
         auto const& shp_mat = loc_asm.getShapeMatrix(int_pt);
-        assert(shp_mat.rows() == nn);
+        assert(shp_mat.cols() == nn);
 
         // copy shape matrix to extrapolation matrix columnwise
-        N.block(0, int_pt, nn, 1) = shp_mat;
+        N.block(0, int_pt, nn, 1) = shp_mat.transpose();
```

Comment:
Maybe it's better to change the whole of N to be transposed. The one can also remove `transpose()` in line 105.